python/cuml/ts/batched_arima.pyx

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+
+import ctypes
+cimport numpy as np
+from libcpp.vector cimport vector
+from libc.stdlib cimport malloc, free
+from libcpp cimport bool
+from libcpp.string cimport string
+cimport cython
+from .batched_kalman import pynvtx_range_push, pynvtx_range_pop
+
+cdef extern from "ts/batched_arima.h":
+  void batched_loglike(double* y, int num_batches, int nobs, int p,
+                       int d, int q, double* params, vector[double]& vec_loglike, bool trans)
+
+
+def batched_loglike_cuda(np.ndarray[double, ndim=2] y, int num_batches, int nobs,
```

Comment:
This also should use input_utils, or at least be using numba arrays instead of numpy